### ISSUE TITLE
Namespace Sass variables

### ISSUE THREAD

**Issue by tawashley** [code-computerlove/slate#3]
Although they are split up into different files i normally prefer to namespace my Sass variables to make their function clear.

Generally along the lines of:

$C-... For colour
$F-... For font stacks
$T-.. For type settings
$MQ-... For media queries
$D-... For dimensions. E.g. $-Gutter: .625rem;
$CS-.. for code snippets. e.g. $CS-Border: $D-BorderWidth solid $C-Grey; For things that are repeatedly used throughout the stylesheets

**Comment by bmds** [comment]
I agree with this one also we have specified within the FE coding standards that we should be doing this:
https://github.com/code-computerlove/frontend-guidelines/blob/master/FE-guidelines-CSS.md#name-spacing

**Comment by code-mattclaffey** [comment]
$E - Element (incase you had a component specific variable.)

**Comment by jtuds** [comment]
This would need to be done on classes too, they're not currently namespaced

**Comment by tawashley** [comment]
@jtuds Think that should be it's own issue. Is there a reference link for name spacing classes? Not very familiar with them myself.

**Comment by tawashley** [comment]
`$mq-` for media query or `$bp` for breakpoint? (I prefer the latter myself)

**Comment by mmacartney10** [comment]
@tawashley going to implement this with a none capitol
